nengo/builder/operator.py

```diff
@@ -201,6 +201,42 @@ def step():
         return step
 
 
+class SlicedCopy(Operator):
+    """Copy from `a` to `b` with slicing: `b[b_slice] = a[a_slice]`"""
+    def __init__(self, a, b, a_slice=Ellipsis, b_slice=Ellipsis,
+                 kind='set', tag=None):
+        assert kind in ['set', 'inc', 'update']
```

Comment:
If there's no code path executing it directly or indirectly, then a unit test should be added to assert its correctness. On the small off-chance that there's a subtle issue/bug with usage, then this may cause pain to the unsuspecting developer down the road who tries to use it assuming that it works. Likewise, future maintenance may take on an unnecessary burden to support something that is never used. So unless it's useful enough to motivate a unit test, it would be less work to omit for now.